Build fails on OSX with Ruby 2.1 against libmaxminddb 0.5.3

```
/usr/local/opt/rbenv/versions/2.1.0/bin/ruby extconf.rb
checking for maxminddb.h... yes
checking for main() in -lmaxminddb... yes
creating Makefile

make "DESTDIR=" clean

make "DESTDIR="
compiling geoip2.c
In file included from geoip2.c:1:
/usr/local/include/maxminddb.h:57: error: expected ‘=’, ‘,’, ‘;’, ‘asm’ or ‘__attribute__’ before ‘mmdb_uint128_t’
/usr/local/include/maxminddb.h:87: error: expected specifier-qualifier-list before ‘mmdb_uint128_t’
geoip2.c: In function ‘locate_by_path’:
geoip2.c:43: warning: passing argument 3 of ‘MMDB_aget_value’ from incompatible pointer type
make: *** [geoip2.o] Error 1

make failed, exit code 2
```

I have zero C skills, but I'm trying to figure this out locally. If you have a quick solution I'd be very grateful.

--- a/extconf.rb
+++ b/extconf.rb
@@ -4,6 +4,8 @@ extension_name = 'geoip2'
 $LDFLAGS << "#{ENV['LDFLAGS']}"
 $CFLAGS << "#{ENV['CFLAGS']}"
 
+RbConfig::MAKEFILE_CONFIG['CC'] = ENV['CC'] if ENV['CC']
+
 if have_header('maxminddb.h') and have_library('maxminddb')
   create_makefile(extension_name)
 else

